CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
-DEVELOPING
+CHANGES IN VERSION 1.12.1 (FROM 1.12.0)
 ===================================
 
-#### General
-* Added `defaultImageTemplateScale` to arrange image comparison logic [#307](https://github.com/appium/appium-base-driver/pull/307). Read [doc](docs/en/advanced-concepts/image-elements.md) for more details.
+Appium 1.12.1 is a patch release that addresses WebView issues caused by Xcode 10.2 (iOS 12.2)
 
 #### iOS
-- Fix `useXctestrunFile` capability for newer Xcode [#903](https://github.com/appium/appium-xcuitest-driver/pull/903)
+- Updated Appium Remote Debugger so that Safari webview tests are compatible with Xcode 10.2 (addresses [issue #12239](https://github.com/appium/appium/issues/12239)) [#118](https://github.com/appium/appium-remote-debugger/pull/118)
+- Fix `screenshotQuality` cap not being forwarded to WebDriverAgent [#907](https://github.com/appium/appium-xcuitest-driver/pull/907)
+- Fix xctestrun file detection when `useXctestrunFile` is true [#903](https://github.com/appium/appium-xcuitest-driver/pull/903)
+- Fix problem with Appium not using cached WebDriverAgent [#909](https://github.com/appium/appium-xcuitest-driver/pull/909)
 
```

Comment:
A new cap has been added to android, which allows to set size or disable the remote cache: https://github.com/appium/appium-android-driver/commit/959e07d84dc9d09a11735872259ac87511d03b85

Reply:
That isn't in this release. This one is just ios/xcui updates.